qa/config/crimson_qa_rgw_num.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
```

Comment:
This file could be named crimson_cpu_num.yaml since it could probably be used by other suites as well (crimson-rados/cephadm once ready).